src/pwr.rs

```diff
@@ -336,7 +336,13 @@ impl PowerMode for StopMode<'_> {
             // Enter Stop mode
             w.pdds().stop_mode();
             // Disable internal voltage regulator
-            w.lpds().set_bit()
+            // Cat 1 devices of stm32l0x1 family do this differently than the rest
+            if cfg!(feature = "stm32l0x1") && cfg!(feature = "io-STM32L021") {
```

Comment:
Shouldn't this just be:

```suggestion
            if cfg!(feature = "stm32l0x1") {
```

The features in `Cargo.toml` specify `io-STM32L021` for anything with `stm32l0x1` so I'm not sure that other feature adds much.

If you'd like to keep it, shouldn't it be

```suggestion
            if cfg!(feature = "stm32l0x1") || cfg!(feature = "io-STM32L021") {
```

instead?

Reply:
I believe you meant to drop `cfg!(feature = "stm32l0x1")` as checking for `cfg!(feature = "io-STM32L021")` would be sufficient If so, the first suggested change above is incorrect and should be:
```
if cfg!(feature = "io-STM32L021") {
```
I was being extra cautious in case someone in the future might mix `"io-STM32L021"` with a non-`"stm32l0x1"` device.  That is why I had the `&&`, meaning this is a "category 1" device of the "stm32l0x1" family.  Just for reference, below is the reference manual section that documents this (see note at bottom):
 
![image](https://user-images.githubusercontent.com/676181/212071441-1be88785-d906-422c-91c5-487e1bc1e44b.png)

I'll go ahead and change and pull #223 as well.

Thanks!

Reply:
That works too :) thanks for the explanation

Reply:
Regarding the io features, see https://github.com/stm32-rs/stm32l0xx-hal#io-features-io- . As far as I'm aware, these features map directly to the device category (but we re-used the terminology from STM32CubeMX). Checking just the io-feature should be safe.